.github/workflows/svc-bip-api-integration-test.yml

```diff
@@ -44,14 +44,10 @@ jobs:
           export -p | sed 's/declare -x //'
 
           ./gradlew :dockerComposeUp
-
-          ./gradlew -p mocks docker
-
+          ./gradlew -p mocks :build
```

Comment:
I think you could also have `:mock-bip-claims-api:build` here instead!

Reply:
added